src/component/AddTodoInput.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react';
+
+import '../App.css';
+import addImg from '../assets/add.png';
+
+class AddTodoInput extends Component {
+
+  constructor() {
+    super();
+
+    this.state = {
+      todoContent: ''
+    };
+  }
+
+  updateStateContent = e => {
+    const todoContent = e.target.value;
+    this.setState({
+      todoContent
+    });
+  };
+
+  checkEnterPressed = e => {
+    if (e.key === 'Enter') {
+      this.addTodo();
+    }
+  };
+
+  addTodo = () => {
+    if (this.state.todoContent !== '') {
+      this.props.addTodoItem(this.state.todoContent);
```

Comment:
We also need to check if the todo content is just white spaces and prevent addition of such todos. So let's trim the content before adding it to the state.